202002_foregrounds_extragalactic_cmb_tophat/README.md

```diff
@@ -0,0 +1,74 @@
+The same folder contains all the configuration files used and the scripts to create SLURM jobs.
+
+## Input components
+
+Each component is saved separately, all maps are in `uK_CMB`, IQU or I, single precision (`float32`)
+
+Dust, synchrotron, free-free and anomalous microwave emission using the "0" models, i.e. `SO_d0`, `SO_s0`, `SO_f0`, and `SO_a0`, see more details in [in the documentation](https://so-pysm-models.readthedocs.io/en/latest/highres_templates.html#details-about-individual-models).
+
+The CIB, KSZ and TSZ models are created from WebSky cosmological simulations, 
+we also include one CMB realization generated with the same cosmological parameters used in the WebSky simulations both unlensed and lensed with the potential from the WebSky simulations,
+see more details in [in the documentation](https://so-pysm-models.readthedocs.io/en/latest/models.html#websky).
+
+All the models listed above will be part of PySM 3 once we upgrade them to nside 8192 and merge them in.
+
+I also created a version of the lensed CMB realization with the dipole component replaced by the [HFI 2018 solar dipole](https://wiki.cosmos.esa.int/planck-legacy-archive/index.php/Map-making#HFI_2018_Solar_dipole)
+
+## Instrument properties
+
+Bandpass for each channel is based on the figures in `s4sim.hardware.config`, PySM executes 10 equally spaced points between the low and the high limit, including both and integrates with the trapezoidal rule. Weighting is performed in `Jy/sr`, the same as PySM 2.
+
+## Available maps
+
+HEALPix maps at high resolution for LAT (nside 4096) and low resolution for SAT (nside 512), these models are deterministic, so we have
+a set for each resolution for all channels. All maps are full-sky.
+
+Reference frame for the maps is **Equatorial**.
+The `ell_max` for the harmonics transform is `3*Nside-1`.
+
+**Location at NERSC**:
+
+    /global/cscratch1/sd/zonca/cmbs4/map_based_simulations/202002_foregrounds_extragalactic_cmb_tophat
+
+The naming convention is:
+
+    {nside}/{content}/{num:04d}/cmbs4_{content}_uKCMB_{telescope}-{band}_nside{nside}_{num:04d}.fits"
+
+where:
+
+* `content` is `[dust, synchtrotron, freefree, ame]` for galactic components and `[ksz, tsz, cib]` for extragalactic components
+* `content` for the available CMB is `[cmb, cmb_unlensed, cmb_lensed_solardipole]`
+*  CIB map for LFL1 and LFS1 are missing
+* `num` is `0`
+* `telescope` is `SAT` or `LAT`
+* `band` is the channel band
+
+## Combined maps
+
+Also created a single set of maps which is the sum of all components to be used as input to some of the TOD simulations (which don't require running PySM on the fly). They are also reordered to NEST (default for TOAST).
+
+* `combined_foregrounds`: sum of `dust`, `synchrotron`, `freefree`, `ame`, `cib`, `ksz`, `tsz`
+* `cmb_lensing_signal`: `cmb_lensed_solardipole` minus `cmb_unlensed`
+* `cmb_lensed_solardipole_nest`: `cmb_lensed_solardipole`, just reordered to HEALPix NEST
+
```

Comment:
@keskitalo here is the plan, I'll add tensor once that is ready.
Do you think some people need only galactic or only extragalactic? should we support that?

Reply:
I can't think of an application that would only want one and not the other. However, that may well be my lack of imagination.

Reply:
ok, let's keep it like this for now